crates/rslint_lexer/src/lib.rs

```diff
@@ -1130,6 +1156,14 @@ impl<'src> Lexer<'src> {
                 tok!(WHITESPACE, self.cur - start)
             }
             EXL => self.resolve_bang(),
+            HAS => {
+                if self.cur == 0 {
+                    self.read_shebang()
+                } else {
+                    self.consume_ident();
+                    tok!(IDENT, self.cur - start)
+                }
```

Comment:
Is this else block logic correct?
I guess that `#` located at the middle of file should be a part of private class fields, but i'm not sure

Reply:
private class fields are a stage 3 proposal, and we don't support proposals yet, so this should be an unexpected token